tests/projects/local-kit/packages/pkg-b/Cargo.toml

```diff
@@ -1,20 +1,19 @@
 [package]
-name = "hello-agent"
+name = "pkg-b"
 version = "0.1.0"
 edition = "2021"
 publish = false
 build = "../build.rs"
 
 [package.metadata.build-package]
-variant-sensitive = false
-source-groups = ["hello-agent"]
+source-groups = []
 
 [lib]
 path = "../packages.rs"
 
 # RPM BuildRequires
 [build-dependencies]
-# None
+core-kit = { path = "../../kits/core-kit" }
```

Comment:
This is one way of having a kit dependency come in: by direct dependency of a package crate on a kit crate.

The other way would be indirectly: a package crate that is only built for a kit crate that depends on another kit crate. Then we could have a rule like "all packages in the k8s kit have access to packages in the core kit because the k8s kit depends on core kit".

But I think that's not supported by the current direction - we need each package to list its kit dependencies.